src/SnackbarContainer.tsx

```diff
@@ -4,37 +4,71 @@ import { makeStyles } from '@material-ui/core/styles';
 import { SNACKBAR_INDENTS } from './utils/constants';
 import { SnackbarProviderProps } from '.';
 
-const useStyle = makeStyles(theme => ({
+const collapse = {
+    container: '& > .MuiCollapse-container',
+    wrapper: '& > .MuiCollapse-container > .MuiCollapse-wrapper',
+};
+
+const useStyle = makeStyles((theme) => ({
     root: {
         boxSizing: 'border-box',
         display: 'flex',
         maxHeight: '100%',
-        maxWidth: '100%',
         position: 'fixed',
-        flexDirection: 'column',
         zIndex: theme.zIndex.snackbar,
         height: 'auto',
         width: 'auto',
-        minWidth: 288,
-        transition: theme.transitions.create(['top', 'right', 'bottom', 'left'], { easing: 'ease' }),
+        transition: 'top 300ms ease 0ms, right 300ms ease 0ms, bottom 300ms ease 0ms, left 300ms ease 0ms, margin 300ms ease 0ms, max-width 300ms ease 0ms',
+        // container itself is invisible and should not block clicks, clicks should be passed to its children 
+        pointerEvents: 'none',
+        [collapse.container]: {
+            pointerEvents: 'all',
+        },
```

Comment:
I found this part is making the snack bar untouchable. `pointer-events` never get unset to `all`. I guess `collapse.container` rules do not apply. Can you check again? Thanks